.github/workflows/_generate-rebase.yaml

```diff
@@ -0,0 +1,52 @@
+# Automatically rebase one branch on top of another; usually staging on top
+# of main after a new package version was published.
+
+name: Rebase branch
+on:
+  workflow_call:
+    inputs:
+      to_head:
```

Comment:
Please add comments to describe the purpose of each input.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/536/commits/7d672a76a53f0899e0bd6c8923aeb21501d4836c.